src/main/java/com/fp3/haras/model/Produto.java

```diff
@@ -27,6 +29,11 @@ public Produto(String nome, int estoque, double pdc, double pdv) {
         this.pdc = pdc;
         this.pdv = pdv;
     }
+    
+    public static Produto getProduto(long id) {
+        Produto p = EntityUtils.select("SELECT c FROM Produto c WHERE Id = " + id, Produto.class).get(0);
```

Comment:
1. The id is param capitalized because it is used as a means to link the object to its version within the database. So through it we match the table object with the object in the database.
2. Yes! Is really working. I made it based on bro Rafa's code